app/assets/stylesheets/colors.css.scss

```diff
@@ -19,6 +19,7 @@ $text-dark-grey: #666666;
 $white: white;
 $black: black;
 $green: #8EDE3D;
+$lightgreen: lighten($green,20%);
```

Comment:
I guess `$light-green` would be more appropriate. (you may call this nitpicking if you want to :-P )